spikeinterface/sorters/launcher.py

```diff
@@ -161,13 +167,20 @@ def run_sorters(sorter_list,
                         shutil.rmtree(str(output_folder))
 
             params = sorter_params.get(sorter_name, {})
+            use_docker = sorter_name in docker_images
+            if use_docker:
+                docker_image = docker_images[sorter_name]
+            else:
+                docker_image = None
+
```

Comment:
Can be done with in one line:

`docker_image = docker_images.get(sorter_name, None)`

Reply:
Done